Lib/test/test_memoryio.py

```diff
@@ -517,6 +517,41 @@ def test_relative_seek(self):
         memio.seek(1, 1)
         self.assertEqual(memio.read(), buf[1:])
 
+    def test_peek(self):
+        buf = self.buftype("1234567890")
+        with self.ioclass(buf) as memio:
+            self.assertEqual(memio.tell(), 0)
+            self.assertEqual(memio.peek(1), buf[:1])
+            self.assertEqual(memio.peek(1), buf[:1])
```

Comment:
You can add tests reading 3 and 5 bytes? It seems like most tests read 1 byte or everything.

Reply:
Absolutely, added now.